### DISCUSSION TITLE
PDF colour is off when using ImageDraw

### DISCUSSION THREAD

**Discussion by bish25** [ImageMagick/ImageMagick#2838] [Development]
Hi, I am having trouble with colour on PDF. black turns into a grey 
```
$draw = new ImagickDraw();
$canvas = new Imagick();
$draw->setFillColor("#000000");
$canvas->annotateImage($draw, $x, $y, 0,"Firstname");
$image->compositeImage($canvas,Imagick::COMPOSITE_ATOP,0, 0);

$image->setImageFormat('pdf');  
header('Content-type: application/pdf');
header('Content-Disposition: inline; filename="preview.pdf"');
echo $image->getImagesBlob();
```

![image](https://user-images.githubusercontent.com/15670486/99411519-9bb9c200-28eb-11eb-8d71-13355775f9cd.png)

**Comment by snibgo** [comment]
Your glyphs are very thin. Perhaps you are seeing only the anti-aliased edges. Try a larger pointsize.

**Reply by bish25** [reply]
Thanks for getting back to me. i have set `$draw->setTextAntialias(false);`  but makes the text like this but still the wrong colour 
![image](https://user-images.githubusercontent.com/15670486/99416730-10dbc600-28f1-11eb-8263-b60c722e51ad.png)
